### ISSUE TITLE
Install fails: Failed to download Chromium r549031 

### ISSUE THREAD

**Issue by robertdebock** [webpro/reveal-md#191]
On some distributions (with different NPM versions) the installation fails with this error:
```
ERROR: Failed to download Chromium r549031! Set "PUPPETEER_SKIP_CHROMIUM_DOWNLOAD" env variable to skip download.
Error: EACCES: permission denied, mkdir '/usr/lib/node_modules/reveal-md/node_modules/puppeteer/.local-chromium'
errno: -13
code: 'EACCES'
syscall: 'mkdir'
path: '/usr/lib/node_modules/reveal-md/node_modules/puppeteer/.local-chromium'
```
You can observe this behavior in a build for an [ansible role](https://travis-ci.org/robertdebock/ansible-role-revealmd/builds/368318766).

The strange thing is that it works on some distributions, and not on others. I'm guessing the NPM version relates to the success or failure.

## Alpine 3.6 (works)
```
# cat /etc/os-release 
NAME="Alpine Linux"
ID=alpine
VERSION_ID=3.6.2
PRETTY_NAME="Alpine Linux v3.6"
HOME_URL="http://alpinelinux.org"
BUG_REPORT_URL="http://bugs.alpinelinux.org"
# npm --version
3.10.10
```
[Example of successful build](https://travis-ci.org/robertdebock/ansible-role-revealmd/jobs/368318769)

## Alpine 3.7 (failed)
```
# cat /etc/os-release 
NAME="Alpine Linux"
ID=alpine
VERSION_ID=3.7.0
PRETTY_NAME="Alpine Linux v3.7"
HOME_URL="http://alpinelinux.org"
BUG_REPORT_URL="http://bugs.alpinelinux.org"
# npm --version
5.5.1
```

[Example of failed build](https://travis-ci.org/robertdebock/ansible-role-revealmd/jobs/368318773)

I have no idea how to fix it. The error was introduced with the release of 2.0.0.

**Comment by webpro** [comment]
Puppeteer is now used instead of PhantomJS for PDF generation, which seems to have a problematic installation. Both reveal-md v2 and puppeteer require at least Node v6.

I have just released v2.0.2 that has puppeteer in the `optionalDependencies`, so the installation of reveal-md itself should no longer fail. Installing this dependency can be skipped altogether using `--no-optional` with the `npm install`.

Obviously PDF generation is unavailable if Puppeteer is missing.

**Comment by robertdebock** [comment]
That's a good enough explanation and solution for me.

In the documentation there is no mention of the requirements, do you want me to add it in a merge request?
